colorama/initialise.py

```diff
@@ -5,9 +5,10 @@
 
 from .ansitowin32 import AnsiToWin32
 
+UNSET = object()
 
-orig_stdout = None
-orig_stderr = None
+orig_stdout = UNSET
```

Comment:
What is the difference between `UNSET` and `None`? Can't we use `None` as "unset"?

Reply:
I was worried that the real sys.stdout might be None (we have a check for that on line 34 / 40 in the diffs of this PR). If that were the case, then when we store sys.stdout in orig_stdout, then orig_stdout will be set to None, and be indistinguishable from the state before 'init' has been called.

Reply:
I agree with @wiggin15. I was worried about stderr but seems that POSIX and Windows require both stdout and stderr.